app/models/test_track/visitor.rb

```diff
@@ -98,6 +98,10 @@ def loaded?
     !offline? && @remote_visitor.present?
   end
 
+  def id_changed?
```

Comment:
Yeah, I think you might want to go with a special name here. Remote won't have as much meaning to a browser of the controller mixin because to them the remote in mind is the client of the controller, not the upstream TT server, so some name that captures that the TT server changed our identity would be hot.